cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -41,6 +41,7 @@
     public static final String EXTERNAL_AUTHENTICATION = "external.authentication";
     public static final String EXPLORE_HTTP_USER_SERVICE = "explore.service";
     public static final String SERVICE_INSTANCE_TABLE_NAME = "cdap.services.instances";
+    public static final String STREAMS_COORDINATOR = "streams.coordinator";
```

Comment:
The stream coordinator is not a service. It shouldn't be define in here. If it is not a shared constant, you can define it in the class where it is used.